openpype/hosts/maya/plugins/create/create_animation.py

```diff
@@ -8,15 +8,13 @@
 )
 
 
-class CreateAnimation(plugin.MayaCreator):
-    """Animation output for character rigs"""
-
-    # We hide the animation creator from the UI since the creation of it
-    # is automated upon loading a rig. There's an inventory action to recreate
-    # it for loaded rigs if by chance someone deleted the animation instance.
-    # Note: This setting is actually applied from project settings
-    enabled = False
+class CreateAnimation(plugin.MayaHiddenCreator):
```

Comment:
@iLLiCiTiT this is not working atm. Am I using this wrong?
The creator is not part of the create context anymore;
```
from openpype.pipeline.create import CreateContext


create_context = CreateContext("maya")
for creator in create_context.creators:
    print(creator)
```